simulations/CC_cycle.py

```diff
@@ -91,14 +93,23 @@ def terminate_check(t, SV, SVdot, return_val, inputs):
         i_data = currents[i]*np.ones_like(solution.values.t)
         cycle_number = int(i+1-equil)*np.ones_like(solution.values.t)
         cycle_capacity = 0.1*solution.values.t*abs(i_data)/3600
+        voltage = solution.values.y.T[-7]
```

Comment:
The `output` function below shows how to get the cell potential more generally:
```python
   phi_ptr = SV_offset + ca.SV_offset+int(ca.SVptr['phi_ed'][-1])
```

Then just replace `-7` with `phi_ptr`.